src/lxc/lxc_init.c

```diff
@@ -258,19 +338,29 @@ int main(int argc, char *argv[])
 		case SIGTERM:
 			if (!shutdown) {
 				shutdown = 1;
+				prevent_forking();
+				if (getpid() != 1) {
+					kill_children(getpid());
```

Comment:
New `glibc` version have thankfully removed their insane `getpid()` userspace cache but this also means this is actually performing two syscalls. I know this is minor but I'd like `getpid()` be called once and record the `pid` in a variable.